README.md

```diff
@@ -80,16 +90,60 @@ NB. the `InstallCommercePackages.ps1` script requires (by default) the Commerce
 PS> docker exec sitecore-commerce-docker_sitecore_1 powershell -Command "C:\Scripts\InstallCommercePackages.ps1"
 ```
 
-After this final installation step commit all changes to the Sitecore Docker image:
+Stop the containers and store the changes, e.g:
+```
```

Comment:
Kunnen we dit niet beter gelijk in een script doen?

Reply:
Ja eigenlijk wel, zal er nog wat liefde aan geven.